src/main/kotlin/dev/stalla/builder/episode/EpisodeBuilder.kt

```diff
@@ -1,3 +1,5 @@
+@file:Suppress("TooManyFunctions")
```

Comment:
I've suppressed the TooManyFunctions in builders since we need at least as many methods in them as there are properties in their data class models.

Reply:
I think we can simply suppress TooManyFunctions and TooManyParameters entirely, they don't really make too much sense when dealing with large models

Reply:
Do you have a favourite config file at hand that we can use? Might be quicker than setting one up from scratch.

Reply:
I'll try to find something